src/extensions/default/Brackets-InteractiveLinter/main.js

```diff
@@ -145,46 +149,56 @@ define(function (require, exports, module) {
             addGutter(currentEditor);
             handleDocumentChange();
         }
-    }
+	}
 
 
-    /**
-     * Function that gets called when Brackets is loaded and ready
-     */
-    function appReady() {
-        // Removes the default Brackets JSLint linter
-        CodeInspection.register("javascript", {
-            name: "interactive-linter-remove-jslint",
-            scanFile: $.noop
-        });
-
-        // Load up plugins and wait til they are done loading before we
-        // register any handlers into Brackets
-        pluginManager().done(function(plugins) {
-            for (var iPlugin in plugins) {
-                linterManager.registerLinter(plugins[iPlugin]);
-            }
-
-            EditorManager.on("activeEditorChange.interactive-linter", setDocument);
-            setDocument(null, EditorManager.getActiveEditor());
-
-            // If the linters change, then make sure to rebind the document with the new linter
-            var lastLinters;
-            preferences.on("change", function() {
-                var editor = EditorManager.getActiveEditor();
-                if (!editor) {
-                    return;
-                }
+  function init() {
```

Comment:
It looks like you've copy/pasted what I did, but neglected to indent it so that it lines up with the surrounding code.  Look at this:

![screen shot 2017-04-21 at 3 16 39 pm](https://cloud.githubusercontent.com/assets/427398/25292790/bb151ac4-26a5-11e7-85ed-7ed93fb8d041.png)

See how the function `appReady` no longer lines up with this?  Everything in this `init` function needs to move over 1 tab (4-spaces) to line up.